postcss.config.js

```diff
@@ -1,8 +1,16 @@
-module.exports = {
+/* eslint-disable no-empty-function */
+module.exports = (ctx) => ({
   plugins: {
+    'postcss-modules': {
+      getJSON: ctx.extractModules || (() => {})
+    },
     'postcss-cssnext': {
       browsers: [`last 2 versions`, `IE > 10`]
     },
+    cssnano: {
+      preset: `default`,
```

Comment:
cssnano does some interesting magic that consolidates styles and tries to look for repeated styles to consolidate.